app/(admin)/layout.tsx

```diff
@@ -31,10 +35,28 @@ const RootLayout = ({
 }): JSX.Element => {
   return (
     <html lang="en" className={GeistSans.className}>
-      <body className="dark:dark bg-background pb-8 px-4 text-foreground xl:pb-0">
+      <body className="dark:dark bg-background text-foreground">
         <ErrorBoundary>
           <AuthContextProvider>
-            <main>{children}</main>
+            <main className="grid grid-cols-adminLayout h-screen">
+              <section className="left-column grid grid-rows-adminLayout border-r-2 border-border">
+                <div className="flex items-center border-b-2 border-border">
+                  <AdminLogo />
```

Comment:
This should encompass the div on line 43 as well. 1 component to house all it's elements.

Reply:
Added to the component